src/main/java/com/qaprosoft/carina/demo/utils/AuthService.java

```diff
@@ -0,0 +1,43 @@
+package com.qaprosoft.carina.demo.utils;
+
+import com.qaprosoft.carina.demo.gui.pages.AutomationPracticeCreateAccountPage;
+import com.qaprosoft.carina.demo.gui.pages.AutomationPracticeSignInPage;
+
+import java.util.Random;
+
+public class AuthService {
+
+    public void signIn(AutomationPracticeSignInPage signInPage, String email, String password) {
```

Comment:
do not use pages as an argument for method. Define needed pages inside methods. In carina util classes you can find how to get driver here